README.md

```diff
@@ -1,134 +1,142 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
+# fairlearn
 
+The project `fairlearn` seeks to enable anyone involved in the development of artificial intelligence (AI) systems to assess their system's fairness and mitigate the observed unfairness. The fairlearn repository contains a Python package and Jupyter notebooks with the examples of usage.
```

Comment:
I would have "`fairlearn` project" instead. Also the second `fairlearn` should be in backquotes too.

Reply:
Done. I'm actually dropping the first backquotes (since then we'd have to use backquotes throughout). Instead, let's stick to backquotes only for the code.